docker/Dockerfile

```diff
@@ -1,17 +1,23 @@
-FROM php:8-buster
+FROM php:7.4-cli-alpine
 
-RUN apt-get -y update && apt-get -y install git zip && \
-    curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer && \
-    chmod +x /usr/local/bin/composer && \
-    pecl install ast-1.0.14 xdebug && \
-    docker-php-ext-enable ast xdebug && \
-    # The pcntl extension is used for speeding up `make phan`
-    docker-php-ext-install pcntl
+ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
 
-# install GRPC
-RUN apt install -y --no-install-recommends zlib1g-dev && \
-    pecl install grpc && \
-    docker-php-ext-enable grpc
+RUN chmod +x /usr/local/bin/install-php-extensions; \
+    apk add --no-cache bash git binutils; \
```

Comment:
I only added git, because it comes with the Debian image and did not know if someone may use it. It's probably not needed, but not that much of an overhead.